samples/posix/uname/src/main.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2023 Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <unistd.h>
+
+#include <sys/utsname.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/shell/shell.h>
+
+int main(void)
+{
+	struct utsname info;
+
+	uname(&info);
+
+	printk("\nPrinting everything in utsname...\n");
+	printk("sysname[%zu]: %s\n", sizeof(info.sysname), info.sysname);
+	printk("nodename[%zu]: %s\n", sizeof(info.nodename), info.nodename);
+	printk("release[%zu]: %s\n", sizeof(info.release), info.release);
+	printk("version[%zu]: %s\n", sizeof(info.version), info.version);
+	printk("machine[%zu]: %s\n", sizeof(info.machine), info.machine);
+
+	return 0;
+}
+
+#define UNAME_KERNEL   (1 << 0)
```

Comment:
@ycsin - sorry - I missed this (was reviewing on mobile).

Please use `BIT(n)` here